public/javascripts/Progress/src/Progress.js

```diff
@@ -57,4 +57,39 @@ function Progress (_, $, difficultRegionIds, userRole) {
         InitializeSubmittedLabels(map, streetParams, 'null', InitializeMapLayerContainer(), data2[0])
         setRegionFocus(map, layers);
     });
+
+    function logWebpageActivity(activity){
```

Comment:
It also might make more sense to just log this on the back-end when we make the update? I'm not totally convinced of that, because maybe we'll want to log more stuff on the front-end here in the future. Your call!

Reply:
I thought about this before, and since I'm sure that we will add more buttons and whatnot to the dashboard, it would be nice to have the logging functionality in Progress.js.